client.go

```diff
@@ -113,7 +116,7 @@ func (c *Client) buildJSONRequest(operation internal.Operation) (*http.Request,
 }
 
 func (c *Client) setRequestHeaders(req *http.Request, desc *internal.Description) error {
-	ua := "OmiseGo/2015-11-06"
+	ua := "OmiseGo/" + libraryVersion
```

Comment:
Can we add API version as well `OmiseAPI/2015-11-06`?

Reply:
It is already there `req.Header.Add("Omise-Version", c.APIVersion)`